### ISSUE TITLE
Investigate Paris transit sources - RATP GTFS file fails on import

### ISSUE THREAD

**Issue by rmglennon** [transitland/transitland#59]
We had tried to import Paris RATP, but it kept failing likely due to memory issues. 

https://transit.land/feed-registry/operators/o-u09-ratp100

A user contacted us through support (ticket 693) asking about the status of this feed, and provided us with helpful additional information regarding the operators in the region. Adding an issue for us to review this.

--

"I was looking at the GTFS, you may want to know that the 
subway in Paris stand for Metro (cover by the RATP), 
the bus was a part of RATP but moved recently to (STIF), 
the RER is a fast public way to move into Paris from the suburb 
the TRAM is also deliver the suburb but with an other standard as the RER

this file may cover everything (FROM RATP) 
http://dataratp.opendatasoft.com/explore/dataset/offre-transport-de-la-ratp-format-gtfs/

The thing that worry me is for the STIF (the bus by day and night in 
Paris and suburb) the data may be better with that source 
https://opendata.stif.info/page/home/

to diversify the sources i found the file specific for the RER A if needed 
http://www.gtfs-data-exchange.com/agency/rer-a/

So basically with RATP and STIF you should cover all Paris and all the 
suburbs around."

**Comment by drewda** [comment]
tagging @irees for future investigation on the RATP import issues

and tagging @stevevance for, in the future, taking a systematic look at this and other Paris-area coverage

**Comment by drewda** [comment]
Paris feed has been successfully imported, now that we have larger servers (https://github.com/mapzen/transitland-internal/issues/19).

There may be some issues conflating stops with OSM. Appears to be a problem with stops from other feeds as well. We'll investigate more under #830.

Open question: Whether RATP is ready for routing on Valhalla.

**Comment by drewda** [comment]
To make RATP's non-standard timezone format work on Valhalla: https://github.com/valhalla/mjolnir/pull/550
